src/theme/typography/default.ts

```diff
@@ -37,10 +37,10 @@ const typography: TypographyOptions = {
         fontSize: '1.3rem'
     },
     body1: {
-        fontSize: '1.25rem'
+        fontSize: '1rem'
```

Comment:
I made these smaller so that the button shows up in the main view on mobile screens... this is a little trickier with the Japanese right now, but it works with the English.